examples/imagenet_cnn/driver.py

```diff
@@ -0,0 +1,82 @@
+import numpy as np
+import ray
+import ray.services as services
+import os
+import tensorflow as tf
+import imagenet
+import argparse
+import boto3
+import random
+
+import functions
+
+
+numofshuffles = 5
+num_workers = 3
+batchnum = 0
+weights = []
+
+parser = argparse.ArgumentParser(description="Parse information for data loading.")
+parser.add_argument("--s3-bucket", default="sparknet", type=str, help="Name of the bucket that contains the image data.")
```

Comment:
There should be no default `s3-bucket` (sparknet only works for us). Also, this field should be required.